pyproject.toml

```
@@ -0,0 +1,46 @@
+[tool.ruff]
+target-version = "py39"
```

```suggestion
target-version = "py311"
```
ATV is on 3.11 already.